sdl2-vga-terminal/src/tui/utils.cpp

```diff
@@ -0,0 +1,48 @@
+#include "utils.hpp"
+#include <stdexcept>
+
+namespace tui
+{
+	std::pair<uint8_t, uint8_t> align3Strings(const std::string& left, const std::string& middle, const std::string right, const uint8_t width)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Function parameter 'right' should be passed by const reference.](https://app.codacy.com/manual/Raffaello/sdl2-vga-terminal/pullRequest?prid=5472420)